seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -345,7 +386,7 @@
       ],
```

Comment:
This challenge description has a mistake. 

References to `elem` should be changed to `num` or vice-versa.

Sorry for all the corrections @no-stack-dub-sack !